adoc/chapters/what_changed.adoc

```diff
@@ -217,8 +217,8 @@ The requirement for {cpp} standard layout for data shared between host
 and devices has been relaxed.  SYCL now requires data shared between
 host and devices to be <<device-copyable>> as defined <<sec::device.copyable>>.
 
-The concept of a <<group>> of <<work-item,work items>> was generalized to include
-<<work-group,work groups>> and <<sub-group,sub groups>>.  A <<work-group>> is represented
+The concept of a <<group>> of <<work-item,work-items>> was generalized to include
+<<work-group,work-groups>> and <<sub-group,sub groups>>.  A <<work-group>> is represented
```

Comment:
```suggestion
<<work-group,work-groups>> and <<sub-group,sub-groups>>.  A <<work-group>> is represented
```

Since you are adding hyphens to this line anyway, let's add this one too.

Reply:
Grand. Thanks!